composer.json

```diff
@@ -19,7 +19,7 @@
     "minimum-stability": "stable",
     "require": {
         "php": ">=7.4",
-        "sequra/integration-core": "v1.0.13",
+        "sequra/integration-core": "dev-fix/CS-5639",
```

Comment:
This should point to a proper version merchants may have at composer.json "minimum-stability" as stable.

Reply:
We will release new core version once the core changes pass your code review.

Reply:
Ok, it is already merged.